main.py

```diff
@@ -1,12 +1,12 @@
-import PyQt5.QtWidgets as qtw
-import PyQt5.QtGui as qtg
-from PyQt5.QtCore import Qt, QPoint, QTimer
-
-import datetime
 from configparser import ConfigParser
+import datetime
 
+from PyQt5.QtCore import QPoint, QTimer, Qt
+import PyQt5.QtGui as qtg
+import PyQt5.QtWidgets as qtw
 from win10toast import ToastNotifier
 
+
 class addTimer(qtw.QDockWidget):

```

Comment:
I can add documentation strings for each of the methods in `addTimer` if you'd like @Wolfmyths.
I created a PR although I probably wouldn't rebase/merge onto base until any changes you want done pertaining to #3 have been integrated into this branch.